Program.cs

```diff
@@ -296,6 +303,65 @@ private int GetPlayerProtection()
             return protection;
         }
 
+        private void fightDescriptionDie( Monster monster )
+        {
+            int dieDescription = new Random().Next(1,6);
+            switch ( dieDescription )
+            {
+                case 1: TypeWriter.WriteLine(new Text($"The {monster.spices} "),
+                                             new Text("stabed you through the hart ", ConsoleColor.DarkRed),
+                                             new Text("and danced on your grave!")); break;
+                case 2: TypeWriter.WriteLine(new Text($"the {monster.spices} "),
+                                             new Text("choped your head off ", ConsoleColor.DarkRed),
+                                             new Text("and took it as a trophy")); break;
+                case 3: TypeWriter.WriteLine(new Text($"The {monster.spices}" ),
+                                             new Text("got you with such a mean "),
```

Comment:
Why put this over 2 lines, you are not changing colour?

Reply:
no idea